service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -32,15 +52,23 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
             }
         }
 
+        ServiceBinding<?> primaryServiceBinding = serviceBindings[0];
         // Bind the service info for the first one passed in
-        binder.bind(ServiceInfo.class).toProvider(new ServiceInfoProvider(serviceBindings[0].serviceInterface()));
+        binder.bind(ServiceInfo.class).toProvider(
+                new ServiceInfoProvider(
+                        primaryServiceBinding.serviceInterface(),
+                        Arrays
+                                .stream(serviceBindings)
+                                .map(ServiceBinding::serviceInterface)
+                                .toArray(Class[]::new)
+                ));
```

Comment:
> Previously, ServiceInfo didn't contain any information about specific sub-services, just the name of the parent service.

IIUC, that was a mistake. See scaladsl's `ServiceInfo`: https://github.com/lagom/lagom/blob/23342691d9cf04e7cb9707848ce8cf5311dcd4df/service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/ServiceInfo.scala#L22-L22 and also compare that to ConductR's mapping of Service+locatableService+ACLS